genapkovl-lima.sh

```diff
@@ -293,6 +293,11 @@ if [ "${LIMA_INSTALL_CRI_DOCKERD}" == "true" ]; then
     cp /home/build/cri-dockerd.license "${tmp}/usr/share/doc/cri-dockerd/LICENSE"
 fi
 
+if [ "${LIMA_INSTALL_CGROUP_CONF}" == "true" ]; then
+    mkdir -p "${tmp}/etc/conf.d"
+    echo 'rc_cgroup_mode=${LIMA_ALPINE_CGROUP_MODE:-hybrid}' >> ${tmp}/etc/conf.d/cgroups
+fi
```

Comment:
```suggestion
if [ -n "${LIMA_CGROUP_MODE}" ]; then
    mkdir -p "${tmp}/etc/conf.d"
    echo 'rc_cgroup_mode=${LIMA_CGROUP_MODE}' >> ${tmp}/etc/conf.d/cgroups
fi
```

Reply:
@jandubois 

Thank you for your review and suggestion. I apologize for my insufficient explanation.

In this PR, I intended to launch the lima machine as `rc_cgroup_mode=unified` when the lima template is set as follows:

```yaml
env:
  LIMA_ALPINE_CGROUP_MODE: unified
```

In other words, this PR creates a file `/etc/conf.d/cgroups` with the content:

```
rc_cgroup_mode=${LIMA_ALPINE_CGROUP_MODE:-hybrid}
```

So that at the time of lima machine startup:

- If the `LIMA_ALPINE_CGROUP_MODE` environment variable is not set in the lima template, cgroup will be (default value) hybrid
- If the `LIMA_ALPINE_CGROUP_MODE` environment variable is set in the lima template, cgroup will be set to that value
This is the modification.

With the above, at machine startup, cgroups can be selected using environment variables, eliminating the need to prepare separate Alpine images for each cgroup mode.

(I included the string `ALPINE_` because it becomes an option at lima startup, but I think it's actually redundant  as you said...)

Based on the above intentions, would you say the modification is appropriate? Or is the modification policy inappropriate?

Reply:
I have uploaded the images, which can be used to test the above behavior, at the following location:

https://github.com/na0x2c6/alpine-lima/releases/tag/v0.2.32-std-cgroup-experimental